app/models/course/assessment/question/text_response.rb

```diff
@@ -121,4 +127,10 @@ def validate_grade
 
     errors.add(:maximum_grade, :invalid_grade)
   end
+
+  def max_attachment_size_defined_if_max_attachments_is_nonzero
+    return unless max_attachments > 0 && !max_attachment_size?
```

Comment:
Let's try to not use unless whenever possible. Personally, I think unless adds another level of complexity when reading code and I dont really agree why ruby enforces it. (https://www.reddit.com/r/ruby/comments/ob8f6k/is_it_just_me_or_is_unless_needlessly_confusing/)

Reply:
done